eng/pipelines/coreclr/templates/run-performance-job.yml

```diff
@@ -90,7 +90,7 @@ jobs:
       displayName: Build scenario tools
       env:
         PERFLAB_TARGET_FRAMEWORKS: netcoreapp3.1
-      condition: and(succeeded(), eq(variables['Agent.Os'], 'Windows_NT'))
+      condition: and(succeeded(), eq(variables['Agent.Os'], 'win'))
```

Comment:
Whatever uses `Agent.OS` shouldn't be changed. Those values are set by Azure DevOps and they return full OS names.